### PR TITLE
Create new Troubleshooting guide

### PR DESCRIPTION
**Pull request by chris1984**

### CODE REVIEW

**Comment by melcorr** [comment]
Hey @chris1984 - can you give me some context for this?

File: guides/doc-Troubleshooting_Guide/common/Makefile [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+$(IMAGES_TS): $(IMAGES)
+	cp $? $(IMAGES_DIR)/
+	pushd "$(IMAGES_DIR)" && for IMG in *.*; do ln -s ../../images/$$IMG ../common/images/$$IMG; done && popd
+	touch $(IMAGES_TS)
+
+$(DEST_HTML): $(SOURCES)
+	asciidoctor -a build=$(BUILD) -a date_mdy="$(DATE_MDY)" -a date_my="$(DATE_MY)" -a docdatetime="$(DATE_MY)" -a stylesheet=$(BUILD).css -a linkcss=true -b html5 -d book --trace -o $@ $<
+
+$(DEST_PDF): $(SOURCES) $(IMAGES)
+	asciidoctor-pdf -a build=$(BUILD) -d book --trace -o $@ $<
```
Comment (lzap) [inline guides/doc-Troubleshooting_Guide/common/Makefile:58]: The `common` directory should have been a symlink to the global common directory.

**Comment by lzap** [comment]
I also don't understand what the intentions are.

**Comment by chris1984** [comment]
Hi @lzap @melcorr this idea was brought on by our Katello Docs day. We didnt have much in the way of a troubleshooting guide, so I picked up the task to make one.

The files that have the actual docs are:

https://github.com/theforeman/foreman-documentation/pull/402/files#diff-c933d1c877e6f95998eb4a6030469f20d8732170179780f1916c9dc4b815d29b

https://github.com/theforeman/foreman-documentation/pull/402/files#diff-ac01103b8c5bd3ba6c30d5c9c7179205965afc578045f93d5e2f70c31c067a91

If I made the guide the wrong way let me know and I can fix stuff

**Review by melcorr — CHANGES_REQUESTED** [review]
Hi @chris1984 
Thanks so much for this! 
I think the main issue is that this PR copies over the entire common directory, when it should have a symlink to that directory. 
When this is fixed, if you could also copy over your files from /topics, and remove the /topics folder, and instead add your new files to /common/modules. You weren't to know this, but it would help if you did. If you've no time, I can fix it after the PR is merged. No big deal at all.

**Pull request closed without merging**